### PR TITLE
add a wrapper detector to D22B

### PR DESCRIPTION
**Pull request by gvardany**
**Description of work.**
This PR does 2 things:
- Wraps the 2 banks of D22B under a dummy component `detector`. This is what is moved during beam center correction, so it is wrong without. There should not be any functional implications, since the dummy component is placed at 0,0,0.
- Removes D11B and D22B from facilities.xml, so IDFs are loaded by file and not by name. This is not to show them in the instrument selector list as additional instruments, since they are just the upgrades of the existing ones.

**To test:**
- [x] Go to the settings menu, make sure there are no D11B and D22B as separate instruments under facility ILL
- [x] Load the new D22B, go to the 4th tab of instrument viewer. Check that in the tree there is a detector component, and inside detector_back and detector_front.

*There is no associated issue.*

*This does not require release notes since support for this instrument is added this release.*

*Reviewer, merge with PR https://github.com/mantidproject/mantidgeometry/pull/179 in geometry.* 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by MathieuTillet — APPROVED** [review]
It works as described, so good to be merged once the tests are ok.

File: instrument/Facilities.xml [hunk 1]
```diff
@@ -684,10 +684,6 @@
     <technique>SANS</technique>
   </instrument>
 
-  <instrument name="D11B">
```
Comment (martyngigg) [inline instrument/Facilities.xml:687]: Can I just double check that this is intentional that these instruments are removed?

Comment (gvardany) [reply instrument/Facilities.xml:687]: Yes, that was the point. The IDFs are there of course, but they shouldn't be listed in facilities as separate from D11 and D22.

Comment (martyngigg) [reply instrument/Facilities.xml:687]: Ah yes I see now. Thanks for clarifying.

**Pull request merged**